auth/src/main/res/values/colors.xml

```diff
@@ -1,11 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <color name="authui_colorPrimary">#3E50B4</color>
-    <color name="authui_colorPrimaryDark">#2F3E9E</color>
-    <color name="authui_colorAccent">#3E50B4</color>
-    <color name="authui_inputTextColor">#FFFFFF</color>
-    <color name="authui_inputTextColorDark">#000000</color>
+    <color name="colorPrimary">#3F51B5</color>
+    <color name="colorPrimaryDark">#303F9F</color>
+    <color name="colorAccent">#FF4081</color>
```

Comment:
I see.  I wasn't thinking it through all the way.  **Before** this change the colors in our theme mattered since we were not leaning on AppCompat, so if the developer did not pass a theme those colors would show up.

Now because we are leaning on AppCompat our theme will almost never show up (unless the dev is not using AppCompat, right?)